assets/src/components/form/inputs/base/option-inputs.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External imports
+ */
+import { isArray } from 'lodash';
+import PropTypes from 'prop-types';
+import { Field } from 'react-final-form';
+
+/**
+ * Internal imports
+ */
+import { InputLabel } from './input-label';
+
+/**
+ * generates the <label><input> pairings for
+ * a set of html checkbox or radio inputs
+ *
+ * @function
+ * @param {string} type
+ * @param {string} name
+ * @param {Object} checkedState
+ * @param {string} htmlId
+ * @param {string} htmlClass
+ * @param {string} labelClass
+ * @param {Array} options
+ * @param {Function} onChange
+ * @param {Function} onClick
+ * @param {string} helpTextID
+ * @param {Object} dataSet
+ * @param {Object} attributes
+ * @return {string} css class describing label size
+ */
+export const OptionInputs = ( {
+	type,
+	name,
+	checkedState,
+	htmlId,
+	htmlClass,
+	labelClass,
+	options,
+	onClick,
+	helpTextID,
+	dataSet,
+	...attributes
+} ) => {
+	delete attributes.initialValue;
```

Comment:
Correct, because React has a hissy fit since `initialValue` does not correspond to a valid HTML attribute when it tries to convert the camelCase prop to something.

A potentially more thorough way to do this would be to store a list of valid HTML attributes and filter out any that are not on that list, but that has its downsides as well (like being kept up to date, etc)

Reply:
Oh yea forgot about that.  I actually submitted a pull request for  Gutenberg that did something similar a while back.